src/pyhf/tensor/jax_backend.py

```diff
@@ -17,8 +18,9 @@ def __init__(self, rate):
         self.rate = rate
 
     def sample(self, sample_shape):
-        return poisson.osp_stats.poisson(self.rate).rvs(
-            size=sample_shape + self.rate.shape
+        tensorlib = jax_backend()
```

Comment:
Getting the `tensorlib` this way is how we've done thing previously inhere, but not sure if there's a smarter way.

Reply:
this isn't the best, since they might have some configurations on the jax backend that are different than the default you're relying on here.